### ISSUE TITLE
.github/PULL_REQUEST_TEMPLATE.md Error

### ISSUE THREAD

**Issue by omkarkd** [education/GitHubGraduation-2021#5232]
Getting this Error, Can anyone help me on this 

Please ensure all changes are contained within the _data/omkarkd/ directory. Invalid file paths:

.github/PULL_REQUEST_TEMPLATE.md

**Comment by ME-ON1** [comment]
hi i saw your commit and .md is not properly formatted and you deleted PRtemplate as well 

try making a new PR to get eligible for Graduation 2021 !! 
The .md file needs to be formatted like this. 
---
name: FULLNAME-OR-NICKNAME # No longer than 28 characters
institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
github_user: YOUR-GITHUB-USERNAME
---

**Comment by elisemoe** [comment]
> Getting this Error, Can anyone help me on this
> 
> Please ensure all changes are contained within the _data/omkarkd/ directory. Invalid file paths:
> 
> .github/PULL_REQUEST_TEMPLATE.md

Looks like your PR is ready to merge! 

There's both an automated and manual review of each PR. The automated review will verify file paths, character counts, etc. of your pull request. Once this check is complete, a "ready to merge" label is applied to your PR. Once this label is present, our team does a manual check to verify that everything meets the event Code of Conduct. This will take us some time to get through everyone but we're working hard to get there! 😁
